src/Commands/Environment/EnvironmentGetEnv.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Copyright (c) Since 2020 Friends of Presta
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file docs/licenses/LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/afl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to infos@friendsofpresta.org so we can send you a copy immediately.
+ *
+ * @author    Friends of Presta <infos@friendsofpresta.org>
+ * @copyright since 2020 Friends of Presta
+ * @license   https://opensource.org/licenses/AFL-3.0  Academic Free License ("AFL") v. 3.0
+ *
+ */
+
+namespace FOP\Console\Commands\Environment;
+
+use FOP\Console\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class EnvironmentGetEnv extends Command
+{
+    private $environmentKeys = [
+        'database_host',
+        'database_port',
+        'database_name',
+        'database_user',
+        'database_password',
+        'database_prefix',
+        'database_engine',
+        'mailer_transport',
+        'mailer_host',
+        'mailer_user',
+        'mailer_password',
+        'secret',
+        'ps_caching',
+        'ps_cache_enable',
+        'ps_creation_date',
+        'locale',
+        'use_debug_toolbar',
+        'cookie_key',
+        'cookie_iv',
+        'new_cookie_key',
+    ];
+
+    protected function configure(): void
+    {
+        $this->setName('fop:environment:get-env')
+            ->setAliases(['fop:environment:getenv'])
```

Comment:
```suggestion
        $this->setName('fop:environment:get-env')
```
the alias is not needed. (it was for backward compat on other commands)